src/main/java/com/nirima/jenkins/plugins/docker/DockerCloud.java

```diff
@@ -361,26 +365,32 @@ int countContainersInProgress() {
                         t.getImage(), label, getDisplayName());
 
                 final CompletableFuture<Node> plannedNode = new CompletableFuture<>();
-                r.add(new NodeProvisioner.PlannedNode(t.getDisplayName(), plannedNode, t.getNumExecutors()));
+                DockerNodeFactory.DockerPlannedNode pn =
+                        NODE_FACTORY.createPlannedNode(t.getDisplayName(), plannedNode, t.getNumExecutors(),
+                                getDisplayName(), t.getImage(), label.getName());
 
+                r.add(pn);
                 final Runnable taskToCreateNewSlave = new Runnable() {
                     @Override
                     public void run() {
                         DockerTransientNode slave = null;
                         try {
-                            // TODO where can we log provisioning progress ?
+                            // TODO where can we log provisioning progress ? // In the CloudMonitor API!
+                            pn.notifyStarted(); // get().onStarted(id);
```

Comment:
Why the commented out code?  What does that signify?
(I generally don't like commented out code - it smells of tech-debt to me - by all means add comments, but preferably not in the form of commented out code that has no context or explanation)

Reply:
This is the code that the other docker plugin used, to make sure I didn't miss any. I can take it out if it is annoying